src/activities/ActivityUsageEntity.js

```diff
@@ -200,12 +200,62 @@ export class ActivityUsageEntity extends Entity {
 		await this._setDate(dateValue, Classes.dates.dueDate, 'dueDate');
 	}
 
+	/**
+	 * @returns {boolean} If F18167-flexible-assignment-dates-ux feature is enabled
```

Comment:
This is unrelated to the assignments, this is for all activities

Reply:
Good point, will update that. I was just testing using assignments.